internal/namespaces/rdb/v1/custom_backup.go

```diff
@@ -233,6 +233,99 @@ func backupListBuilder(c *core.Command) *core.Command {
 	return c
 }
 
+func backupExportDisplayBuilder(i interface{}, opt *human.MarshalOpt) (string, error) {
+	version := i.(rdb.DatabaseBackup)
+
+	type customDisplayBackup struct {
+		ID           string                   `json:"ID"`
+		InstanceID   string                   `json:"instance_ID"`
+		DatabaseName string                   `json:"database_name"`
+		Name         string                   `json:"name"`
+		Status       rdb.DatabaseBackupStatus `json:"status"`
+		Size         *scw.Size                `json:"size"`
+		ExpiresAt    *time.Time               `json:"expires_at"`
+		CreatedAt    *time.Time               `json:"created_at"`
+		UpdatedAt    *time.Time               `json:"updated_at"`
+		InstanceName string                   `json:"instance_name"`
+		IsExported   bool                     `json:"is_exported"`
+		Region       scw.Region               `json:"region"`
+		SameRegion   bool                     `json:"same_region"`
+	}
+
+	opt.Sections = []*human.MarshalSection{
```

Comment:
I would not create a section per field as it make the output very long and does not create much hierarchical visualization